internal/statestore/testing/public.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testing
+
+import (
+	"time"
+
+	"github.com/alicebob/miniredis"
+	"github.com/spf13/viper"
+	"open-match.dev/open-match/internal/config"
+)
+
+// NewRedisForTesting create a new in-memory miniredis for testing
+func NewRedisForTesting(cfg *viper.Viper) (config.View, func(), error) {
```

Comment:
This should not expose the closer - any component using this cfg to build a StateStore object should be calling .Close() on the public state store method. 

Do consider not reusing this helper for redis_test.go.